markdownlint/TOP005_blanksAroundMultilineHtmlTags/tests/TOP005_test.md

```diff
@@ -0,0 +1,101 @@
+### Introduction
+
+This file should not be flagged with any errors.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+Valid div due to each tag being surrounded by blank lines.
+
+</div>
+
+#### Custom section
+
+<div>Valid single-line div</div>
+
+<div>Valid single-line div</div>Might even have other <span>paragraph</span> content with it.
+
+<div class="lesson-content__panel" markdown="1">
+1. Opening tag invalid due to not being surrounded by blank lines.
+2. Closing tag is, so is valid.
+3. Notice how non-lazy numbering does not flag a lint error, as the list content is part of the same `html_block` token, and not as their own separate tokens to be linted.
+
+</div>
+
+Non-empty line
+<div class="lesson-content__panel" markdown="1">
+
+1. Opening tag invalid due to not being surrounded by blank lines.
+2. Closing tag invalid due to not being surrounded by blank lines.
```

Comment:
Can we just update the rest of this file to ensure that *only* this TOP005 rule gets flagged? Just kinda adds clutter when having to sift through the non-blank lines errors.